### DISCUSSION TITLE
Thinking About Additional Report Output Methods

### DISCUSSION THREAD

**Discussion by AlexSim93** [AlexSim93/pull-request-analytics-action#12] [Q&A] [answered]
I'm considering adding more ways to output reports in pull-request-analytics-action. Currently, there's an option to create a new issue for each report. You can also get a markdown string or a data collection. I see potential in generating report artifacts, such as PDFs or HTML pages. Additionally, within GitHub, reports could be integrated into other places like pull requests, comments, etc. Another possibility is integrating with external services like Notion, Slack, and others.

**Comment by razbensimon** [comment]
integrate Notion, Slack could be amazing

**Comment by AlexSim93** [comment, marked as answer]
I have thought about it. I see it as combination of different actions, because it might be useful not only for reports. I am not sure there are actions that covers all cases, but some of them already created.

For example, it is possible to generate markdown and use [this](https://cli.github.com/manual/gh_pr_comment) to create comment in Pull Request. The same approach(different commands or actions) might be used to create markdown or html file as an artifact.

Do you need some particular way for reporting metrics? I just checked some of your proposals, probably some of them don't have solutions yet.
